vms/authentication/templates/authentication/login.html

```diff
@@ -62,6 +62,11 @@
                 </div>
             </div>
         {% endif %}
+        {% if form.errors %}
+        <div class="alert alert-danger">
+            "Your username and password didn't match. Please try again."
+        </div> <!-- alert alert-danger -->
```

Comment:
@amruthasangeeth When I enter wrong username or/and password I get [this](https://gyazo.com/f3a71910af377d221985ef9f657926e6). How can I get the above error on the UI?